app/Actions/DisplayHelpScreen.php

```diff
@@ -10,28 +10,35 @@ class DisplayHelpScreen
 
     protected $commands = [
         'help' => 'Display this screen',
-        'edit-config' => 'Edit config file',
-        'edit-after' => 'Edit "after" file',
+        'edit-config' => 'Edit "~/.lambo/config" file',
+        'edit-after' => 'Edit "~/.lambo/after" file',
+        'new' => 'Scaffold a new Laravel application',
     ];
 
     public function __invoke()
     {
         app('console-writer')->newLine();
-        app('console-writer')->text('<comment>Usage:</comment>');
-        app('console-writer')->text("  lambo new myApplication [arguments]\n");
         app('console-writer')->text('<comment>Commands (lambo COMMANDNAME):</comment>');
-
         foreach ($this->commands as $command => $description) {
             $spaces = $this->makeSpaces(strlen($command));
             app('console-writer')->text("  <info>{$command}</info>{$spaces}{$description}");
         }
-
         app('console-writer')->newLine();
-        app('console-writer')->text('<comment>Options (lambo new myApplication OPTIONS):</comment>');
-
+        app('console-writer')->text('<comment>Usage:</comment>');
+        app('console-writer')->text('  lambo help');
+        app('console-writer')->text('  lambo [common options] edit-config [--editor=<editor>]');
```

Comment:
What are the common options for edit-config and edit-after?

Reply:
I'm not sure if I love adding help the other commands here.. none of them need to define how we're running those commands, right? And this section is a bit more about instructing how primary usage should be?

Reply:
OK, so I just looked at the common options. Would anyone really pass any of those options to edit-config or edit-after, do you think?

Reply:
I agree that it makes sense to remove the help command but I think `edit-config` and `edit-after` should stay as they both use the `--editor` flag and that isn't doccumented anywhere.

I was going for pendantic accuracy :laughing: but if you prefer instructions for primary usage (which is better tbh) then lets remove the common options from `edit-config` and `edit-after`.

I think we can go a step further and combine/simplfy things like this:

![image](https://user-images.githubusercontent.com/1974648/114390203-dcadbe00-9b8d-11eb-8b78-cd194dcedb5d.png)

What do you think?

Reply:
I've removed the common options.